docs/rows.md

```diff
@@ -29,7 +29,7 @@ An example usage follows.
 
 ## How to filter rows
 
-1. Press `s` or `t` on the rows to filter.
```

Comment:
I can see how this would be confusing, thanks for pointing it out. How about `on the rows to be filtered`?

Reply:
`on the rows to be filtered` even better! 🙏